core/src/main/java/io/aiven/klaw/service/UtilControllerService.java

```diff
@@ -599,6 +602,8 @@ public Map<String, String> getBasicInfo() {
 
     if ("true".equals(ssoEnabled)) {
       resultBasicInfo.put("ssoServerUrl", ssoServerLoginUrl);
+    } else if (authenticationType.equals("azuread")) {
```

Comment:
```suggestion
    } else if ("azuread".equals(authenticationType)) {
```
it's better to flip args

Reply:
Agree, done in all applicable instances too.